aiohttp_sse/helpers.py

```diff
@@ -1,43 +1,62 @@
-from collections.abc import Coroutine
+from types import TracebackType
+from typing import (
+    Any,
+    AsyncContextManager,
+    Coroutine,
+    Generator,
+    Optional,
+    Type,
+    TypeVar,
+)
 
+T = TypeVar("T", bound=AsyncContextManager["T"])
 
-class _ContextManager(Coroutine):
+
+class _ContextManager(Coroutine[T, None, T]):
     __slots__ = ("_coro", "_obj")
 
-    def __init__(self, coro):
+    def __init__(self, coro: Coroutine[T, None, T]) -> None:
         self._coro = coro
-        self._obj = None
+        self._obj: Optional[T] = None
 
-    def send(self, arg):
+    def send(self, arg: Any) -> T:
         return self._coro.send(arg)  # pragma: no cover
 
-    def throw(self, arg):
-        return self._coro.throw(arg)  # pragma: no cover
+    def throw(self, *args) -> T:  # type: ignore[no-untyped-def]
```

Comment:
Can't manage with correct typing, but it seem it's not necessary here

Reply:
This class itself is nasty and should probably be removed, if possible (not in this PR though). We have a bug in aiohttp that may be caused by this same wrapper.